src/main/java/org/dependencytrack/model/VulnerableSoftware.java

```diff
@@ -173,6 +182,10 @@ public class VulnerableSoftware implements ICpe, Serializable {
     @Column(name = "UUID", jdbcType = "VARCHAR", length = 36, allowsNull = "false")
     private UUID uuid;
 
+    @Persistent(mappedBy = "vulnerableSoftware", defaultFetchGroup = "true")
```

Comment:
I noticed that the mirroring became noticeably slower with this PR. Calls to `getVulnerableSoftwareByCpe23` took way more time, and ended up accounting for ~85% of the time spent in the mirroring task:

![image](https://user-images.githubusercontent.com/5693141/199222872-5a00e723-2c52-49f2-b758-a8bd880f8c3a.png)

For comparison, with the current `master`, it's only ~20%:

![image](https://user-images.githubusercontent.com/5693141/199223184-beaf36c2-14b8-4531-9cf3-2006ab1cea58.png)

It looks like the culprit here is that `affectedVersionAttributions` is in the default fetch group (probably causing expensive `JOIN`s or additional queries behind the scenes). Removing the `defaultFetchGroup = "true"` part moves the time spent back to ~20%:

![image](https://user-images.githubusercontent.com/5693141/199223600-88ec79f5-7be5-401b-ae26-87308f04dc20.png)

Because the attributions are fetched ad-hoc via `getAffectedVersionAttribution` anyway, maybe we can remove this field altogether, or make it `transient` if we need to populate it temporarily? See `Vulnerability#findingAttribution` for reference.